debug.Dockerfile

```diff
@@ -0,0 +1,44 @@
+FROM golang:1.25-alpine AS builder
```

Comment:
After this PR is merged https://github.com/open-telemetry/opentelemetry-ebpf-instrumentation/pull/521

I will rebase to remove this file from this PR.